### PR TITLE
Deploy Azure ASM gem from travisCI

### PR DESCRIPTION
**Pull request by vishrutshah**
@veronicagg @devigned 

Could you please review this change to release Azure ASM gem from travis?

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+deploy:
+  provider: rubygems
+  api_key:
+    secure: h1O/CbvwR2fsS013v7QAlW8qLIXrMDED9+HPDi5fM5o/Q/PU7uJTRI3Aold76xzXJj0eDloNIXlocdiYaJ3fh4u4UYv1HjTIW5yz9L30fjhuuWw7lMvBUHz0eYUTPWwWEZvp2aJY07C5KVRn+0sjZs9aKTUUIfFtHsW3uSZZpfY=
+  gemspec: service_management/azure/azure.gemspec
+  gem:
+    master: azure
+  on:
+    tags: true
+    repo: Azure/azure-sdk-for-ruby
```
Comment (veronicagg) [inline .travis.yml]: do we need to specify the repo?

**Comment by veronicagg** [comment]
@vishrutshah Left a comment, rest looks good to me!

File: .travis.yml (hunk 1 above)
Comment (vishrutshah) [inline .travis.yml]: I think it's optional but I used `travis setup rubygems` command to set it up and it asked me to specify.

Comment (veronicagg) [inline .travis.yml]: Sounds good. Thanks!

**Comment by vishrutshah** [comment]
@veronicagg Thanks for the review !

**Pull request merged**